lamp.go

```diff
@@ -2308,6 +2308,94 @@ func deleteScheduleOverrideCommand() gcli.Command {
 	return cmd
 }
 
+func getOnCallsCommand() gcli.Command {
+	commandFlags := append([]gcli.Flag{
+		gcli.StringFlag{
+			Name:  "id",
+			Usage: "ID of the schedule",
+		},
+		gcli.StringFlag{
+			Name:  "name",
+			Usage: "Name of the schedule",
+		},
+		gcli.BoolFlag{
+			Name: "flat, f",
+			Usage: "If the response should be flat",
+		},
+		gcli.StringFlag{
+			Name: "atTime",
```

Comment:
Should this be `date`? I agree `atTime` is more descriptive, but we use a different field in the API: https://docs.opsgenie.com/docs/who-is-on-call-api#get-on-calls

Reply:
yeh, just wanted it to be more friendly to understand